### DISCUSSION TITLE
How to add an addon only in dev?

### DISCUSSION THREAD

**Discussion by afgallo** [aws/copilot-cli#3570] [Q&A] [answered]
Hi,

I'd like to have an addon deployed only in my development environment. Is this possible with Copilot Addons?

Thanks 🙏🏻

**Comment by afgallo** [comment, marked as answer]
I think I may have found an answer through conditions:

```
Conditions:
  CreateResource: !Not [!Equals [!Ref Env, prod]]
```

**Comment by ColeDCrawford** [comment]
@afgallo could you post the rest of the addons YML file? I'm trying to do the same but seem to be getting some of the syntax wrong.
